### PR TITLE
some eZ Platform 3 change

### PR DESCRIPTION
**Pull request by macintoshplus**
* change version required in `composer.json` file
* add controller tag, `autowire` and `autoconfigure`
* fix twig `getFunction` return check on error
* fix `TranslatorInterface` change beetwen Symfony 3.4 and Symfony 5.0

### CODE REVIEW

File: src/Resources/config/services.yaml [hunk 1]
```diff
@@ -6,13 +6,21 @@ services:
 
     CodeRhapsodie\EzDataflowBundle\Controller\DashboardController:
         public: true
+        autoconfigure: true
+        autowire: true
```
Comment (jeremycr) [inline src/Resources/config/services.yaml]: Services from reusable bundles should not use autowiring or autoconfiguration.

https://symfony.com/doc/current/bundles/best_practices.html#services

File: src/Twig/EzTabRenderExtension.php [hunk 2]
```diff
@@ -22,19 +21,17 @@ public function getFunctions()
         ];
     }
 
-    public function crAddEzTabs(Environment $twig, string $param1, array $options = [], string $template = '') {
+    public function crAddEzTabs(Environment $twig, string $param1, array $options = [], string $template = '')
+    {
         $function = $twig->getFunction('ez_platform_tabs');
-        if ($function !== false) {
+        if (false !== $function && null !== $function) {
```
Comment (jeremycr) [inline src/Twig/EzTabRenderExtension.php]: Wouldn't `instanceof` be a more appropriate test in that case ?

Comment (macintoshplus) [reply src/Twig/EzTabRenderExtension.php]: Changed

File: src/Resources/config/services.yaml (hunk 1 above)
Comment (macintoshplus) [reply src/Resources/config/services.yaml]: changed

**Review by jeremycr — APPROVED** [review]

**Pull request merged**